source/db/wifi_db.c

```diff
@@ -408,6 +408,12 @@ static int init_vap_config_default(int vap_index, wifi_vap_info_t *config,
         cfg.u.bss_info.beaconRate = WIFI_BITRATE_6MBPS;
         strncpy(cfg.u.bss_info.beaconRateCtl,"6Mbps",sizeof(cfg.u.bss_info.beaconRateCtl)-1);
         cfg.vap_mode = wifi_vap_mode_ap;
+        /*TODO: Are values correct?  */
+        cfg.u.bss_info.mld_info.common_info.mld_enable = 1;
+        cfg.u.bss_info.mld_info.common_info.mld_id = 1;
+        cfg.u.bss_info.mld_info.common_info.mld_link_id = 1;
+        cfg.u.bss_info.mld_info.common_info.mld_apply = 1;
+        strcpy(cfg.u.bss_info.mld_info.common_info.mld_addr, "11:11:11:11:11:11");
```

Comment:
I assume it's default value updated later somewhere in code.
So maybe it would be better to use empty string "" or some special string like "de:ad:be:ef:00:00" to
point that this value is inccorect.